### ISSUE TITLE
Inform the users that they can edit the details of the image and addition of details

### ISSUE THREAD

**Issue by RishikaBhatia17** [tonythomas01/gdrive-to-commons#35]
After the user selects the pictures, they can view the list of images they have selected along with their title, description and option to unselect.
But if a normal user uses this tool, it is difficult for them to find out that they can edit the title and add the description and this might be a disadvantage for the user and the feature is not being utilised by the user.
![image (2)](https://user-images.githubusercontent.com/46782283/76368406-265a2280-6356-11ea-8b1b-784f01f692d2.png)
Also as mentioned already,
More metadata fields can be added to the image. The important metadata for a image might be:
Date image is created
Categories
Captions
License
Image type
Width and height of the image
Hashtags for the image etc.

The expected solution for the above mentioned issue is:
The user should be informed that they can edit the details of the image by giving user interface instructions and the important metadata fields can also be added.

**Comment by tonythomas01** [comment]
This is true though. Can you start by creating a phabricator task with the same details  ? We do not want people to create bugs here and lose context.
